react/week2/my-app/src/Components/TodoItem.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+
+class TodoItem extends Component {
+  constructor() {
+    super();
+
+    this.onMarkItemComplete = (event) => {
+      this.props.markItemComplete(this.props.id);
+    };
+
+    this.onDeleteItem = (event) => {
+      this.props.deleteItem(this.props.id);
+    };
+  }
```

Comment:
So here you can just declare the functions inside the class, you don't need the constructor.

```suggestion

    onMarkItemComplete = (event) => {
      this.props.markItemComplete(this.props.id);
    }

    onDeleteItem = (event) => {
      this.props.deleteItem(this.props.id);
    }
```